src/run_benchmark_test.py

```diff
@@ -18,9 +20,11 @@ def main() -> int:
         Entry point for Benchmark Test with bundle manifest, config file containing the required arguments for running
         benchmarking test. Will call out in test.sh with benchmark as argument
     """
+    manifest: Union[BundleManifest, BuildManifest]
     benchmark_args = BenchmarkArgs()
     console.configure(level=benchmark_args.logging_level)
-    manifest = BundleManifest.from_file(benchmark_args.bundle_manifest)
+    manifest = BundleManifest.from_file(benchmark_args.bundle_manifest) if not benchmark_args.min_distribution else \
+        BuildManifest.from_file(benchmark_args.bundle_manifest)
```

Comment:
I think you can just combine these two?

manifest: Union[BundleManifest, BuildManifest] = <> ?